index.html

```diff
@@ -253,15 +254,17 @@ <h3>
         </h3>
         <p>
           Two events are used to communicate pointer lock state change or an
-          error in changing state. They are named <dfn>pointerlockchange</dfn>
-          and <dfn>pointerlockerror</dfn>. If pointer lock is entered or exited
+          error in changing state. They are named <dfn class=
+          "event">pointerlockchange</dfn> and <dfn class=
+          "event">pointerlockerror</dfn>. If pointer lock is entered or exited
           for any reason a {{Document/pointerlockchange}} event must be sent.
         </p>
         <p>
           The [=user agents=] must deliver these events by <a data-lt=
           "queue a task">queuing a task</a> to [=fire an event=] of the
-          appropriate name with its `bubbles` attribute [[DOM]] set to false to
-          the pointer lock [=Event/target=] element's [=Node/node document=].
+          appropriate name with its {{Event/bubbles}} set to false to the
```

Comment:
Same here... stuff that happens in algorithms shouldn't be described outside of those algorithms.

Reply:
Done, remove this and leave "Refer to algorithm of [[[#requestPointerLock]]] for detail."